src/txn/txn_rollback_to_stable.c

```diff
@@ -680,7 +607,12 @@ __rollback_abort_row_ondisk_kv(
           __wt_timestamp_to_string(vpack->tw.start_ts, ts_string[1]), prepared ? "true" : "false",
           __wt_timestamp_to_string(rollback_timestamp, ts_string[2]), vpack->tw.start_txn);
         if (!F_ISSET(S2C(session), WT_CONN_IN_MEMORY))
-            return (__rollback_row_ondisk_fixup_key(session, page, rip, rollback_timestamp, true));
+            if (rip != NULL)
```

Comment:
I think we don't need two function flows here. Pass all the provided options to the next function. The flow is decided internally based on `cip` or `rip` is not NULL.

Reply:
Yeah, that makes more sense.